docs/managed-datahub/slack/saas-slack-setup.md

```diff
@@ -0,0 +1,174 @@
+
+- Click **Install to Workspace**
+- It will show you permissions the Slack App is asking for, what they mean and a default channel in which you want to add the slack app
+    - Note that the Slack App will only be able to post in channels that the app has been added to. This is made clear by slack’s Authentication screen also.
+- Select the channel you'd like notifications to go to and click **Allow**
+- Go to DataHub App page
+    - You can find your workspace's list of apps at https://api.slack.com/apps/
+
+## Generating a Bot Token
+
+- Go to **OAuth & Permissions** Tab
+
+![](https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/integrations/slack/slack_oauth_and_permissions.png)
+
+Here you'll find a “Bot User OAuth Token” which DataHub will need to communicate with your slack through the bot. 
+In the next steps, we'll show you how to configure the Slack Integration inside of Acryl DataHub.
+
+## Configuring Notifications
+
+> In order to set up the Slack integration, the user must have the `Manage Platform Settings` privilege. 
+
+To enable the integration with slack
+- Navigate to **Settings > Integrations**
+- Click **Slack**
+- Enable the Integration 
+- Enter the **Bot Token** obtained in the previous steps
+- Enter a **Default Slack Channel** - this is where all notifications will be routed unless 
+- Click **Update** to save your settings
+
+<img width="60%" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/integrations/slack/slack_add_token.png"/>
+
+To enable and disable specific types of notifications, or configure custom routing for notifications,  start by navigating to **Settings > Notifications**.
+To enable or disable a specific notification type in Slack, simply click the check mark. By default, all notification types are enabled.
+To customize the channel where notifications are send, click the button to the right of the check box.
+
+<img width="70%" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/integrations/slack/slack_channel.png"/>
+
+If provided, a custom channel will be used to route notifications of the given type. If not provided, the default channel will be used. 
+That's it! You should begin to receive notifications on Slack. Note that it may take up to 1 minute for notification settings to take effect after saving.  -->
+
+## Sending Notifications
+
+For now we support sending notifications to
+- Slack Channel Name (e.g. `#troubleshoot`)
+- Slack Channel ID (e.g. `C029A3M079U`)
+- Specific Users (aka Direct Messages or DMs) via user ID 
+
+By default, the Slack app will be able to send notifications to public channels. If you want to send notifications to private channels or DMs, you will need to invite the Slack app to those channels.
+
+## How to find Team ID and Channel ID in Slack
+***Note** we recommend just using the Slack channel name for simplicity (e.g. `#troubleshoot`)*
```

Comment:
let's use docusaurus admonition for this part 
<img width="737" alt="image" src="https://github.com/user-attachments/assets/b499aa1e-898d-4ea0-9b00-3520270b5e6a">

```suggestion
:::note 

we recommend just using the Slack channel name for simplicity (e.g. `#troubleshoot`)

:::
```

Reply:
tyyy